Fix image TV display via hash

Ruslan-Aleev opened the pull request:
### What does it do?
Same as in https://github.com/modxcms/revolution/pull/15731, but for 3.x branch

### Related issue(s)/PR(s)
https://github.com/modxcms/revolution/pull/15731

Ruslan-Aleev committed:
Fixed display of image in TV through hash

Ibochkarev commented on core/src/Revolution/Processors/Element/TemplateVar/Renders/mgr/input/image.class.php:
```diff
@@ -22,7 +22,7 @@
  */
 class modTemplateVarInputRenderImage extends modTemplateVarInputRender {
-    public function process($value,array $params = []) {
+    public function process($value, array $params = []) {
```
```suggestion
    public function process($value, array $params = [])
    {
```

Ruslan-Aleev committed:
Update core/src/Revolution/Processors/Element/TemplateVar/Renders/mgr/input/image.class.php

Co-authored-by: Bochkarev Ivan <ivanx86@gmail.com>

Jako commented:
This has to be added too: https://github.com/tanzirev/revolution/commit/aae1ea2329de2712abfb4010afe5e86042e1a11a

Jako committed:
Fix phpThumb private parameter issue

Fix warning: Detected attempt of using private parameter `version` (for internal usage) of phpThumb that not allowed and insecure
